app/views/helpers/HtmlTitle.scala

```diff
@@ -27,8 +27,12 @@ object HtmlTitle {
     (isInvalidInput, isErrorPage, isAgent) match {
       case (false, true, _) => messages("htmlTitle.errorPage", h1Text)
       case (true, false, _) => messages("htmlTitle.invalidInput", h1Text)
-      case (_, _, true) => messages("htmlTitle.agent")
-      case (_, _, _) => messages("titlePattern.serviceName.govUk", h1Text)
+      case (_, _, true) => h1Text + " - " + messages("htmlTitle.agent") + " - " + messages("base.govUk")
+      case (_, _, _) => h1Text + " - " + messages("titlePattern.serviceName.govUk") + " - " + messages("base.govUk")
     }
   }
+
+  def messageTitle(serviceName: String, pageTitle: String)(implicit messages: Messages): String = {
+    pageTitle + " - " + messages(serviceName) + " - " + messages("base.govUk")
+  }
```

Comment:
I think this could be simplified by renaming titlePattern.serviceName.govUk in messages to htmlTitle, then the value could be {0} - Manage your Income Tax updates - GOV.UK, 0 being the h1Text param. And so on for agent?

Reply:
Especially as this creating different functionality for agent v individual where the type of page (normal - no errors/invalid input) is the same